kolibri/core/assets/src/views/core-base.vue

```diff
@@ -110,6 +115,7 @@
       },
     },
     mounted() {
+      this.updateDocumentTitle(this.title);
```

Comment:
The wrapping is straight forward, we can easily invoke vue intl functionality anywhere, it's extracting the strings automatically (which is not perfect even in our current system) that's harder.